.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: ble.sh build
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  builds:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Package
+        run: |
+          mkdir ble.sh
```

Comment:
I suppose this line is unneeded because it will be created by `make install` in the following line after all.
```suggestion
```